packages/core/src/App/Components/Layout/Header/signup-button.jsx

```diff
@@ -1,19 +1,46 @@
-import PropTypes from 'prop-types';
 import React from 'react';
+import PropTypes from 'prop-types';
 import { Button } from '@deriv/components';
-import { redirectToSignUp } from '@deriv/shared';
+import { redirectToSignUp, mobileOSDetectAsync } from '@deriv/shared';
```

Comment:
It should also support Safari in desktop

Reply:
Done